### ISSUE TITLE
Sort out `pypdf2` and `pdfcropmargins` issue 

### ISSUE THREAD

**Issue by oscarlevin** [PreTeXtBook/pretext-cli#289]
Apparently this is still a problem (https://groups.google.com/g/pretext-support/c/wLnnb_3KJCE).  It appears that forcing the older version of pypdf2 as a dependency works, although there are conflicts.

**Comment by oscarlevin** [comment]
Today someone ran into possibly this issue.  The result of the output (even under debug) ends with 

```
cropping f-arrows.pdf to cropped-f-arrows.pdf
critical: Not encrypted file
Failed to generate latex-image elements.  Exiting...
```

**Comment by oscarlevin** [comment]
I can confirm that pypdf2 2.5.0 does not work with pdfcropmargins 1.0.8.  However, reverting to pypdf2 2.3.1 does, as apparently does 1.26.  If possible, we can simply require 2.3.1, if pip is smart enough to downgrade that when installing pretext.

**Comment by StevenClontz** [comment]
We discussed today: let's cap out `pdfcropmargins` at `1.0.8` and pypdf2 at 2.3.1.

**Comment by oscarlevin** [comment]
It looks like `pdfcropmargins` fixed the issue in 1.0.9, so I have specified only patches above 1.0.9 and above 2.5.0 for `pypdf2`, in 713b4ae.
